src/algorithms/digi/SiliconTrackerDigi.cc

```diff
@@ -52,8 +52,8 @@ eicrecon::SiliconTrackerDigi::produce(const std::vector<const edm4hep::SimTracke
 
 
         double edep = sim_hit->getEDep();
-        if (edep * units::keV < m_cfg.threshold) {
-            m_log->debug("  edep is below threshold of {:.2f} [keV]", m_cfg.threshold / units::keV);
+        if (edep / units::keV < m_cfg.threshold) {
+            m_log->debug("  edep is below threshold of {:.2f} [keV]", m_cfg.threshold);
```

Comment:
```suggestion
        if (edep < m_cfg.threshold) {
            m_log->debug("  edep is below threshold of {:.2f} [keV]", m_cfg.threshold / units::keV);
```

Reply:
Also remove `namespace units = TGeoUnit;` above

Reply:
cc #577

Reply:
Hi. I think the changes @veprbl are not correct if we put the number "5" in the SiliconTrackerDigiConfig.h file for the threshold. In that case, we need to convert edep from GeV to keV (or m_cfg.threshold from keV to GeV) before doing the comparison.

Can we also set the threshold in SiliconTrackerDigiConfig.h to 5*units::keV? Then it would be consistent with the changes Dmitry made.

Reply:
Yes, see the suggestion above. @DraTeots marked it as resolved for unknown reason.

Reply:
Ok. I see it now. Thanks. 

@blianggilman if you think that all looks reasonable, you can select the "commit suggestion" for both the SiliconTrackerDigiConfig.h and the SiliconTrackerDigi.cc files.